ui/dashboards/src/views/DashboardApp.tsx

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Box } from '@mui/material';
+import { combineSx } from '@perses-dev/components';
+import { useState } from 'react';
+import { VariableList, Dashboard } from '../components';
+import AddPanel from '../components/AddPanel/AddPanel';
+import { DashboardToolbar } from '../components/DashboardToolbar';
+import { useDashboard } from '../context';
+import { ViewDashboardProps } from './ViewDashboard';
+
+export const DashboardApp = (props: ViewDashboardProps) => {
```

Comment:
nit but this seems a little weird to name this DashboardApp since there is the separate perses-dev/app package, would something like ‘DashboardContent.tsx’ or just ‘Dashboard.tsx’ work (can the old Dashboard component be deleted now btw?)

Reply:
nvm DashboardContent isn't great either since it would be outside of Dashboard.tsx, can't think of anything better at the moment

Reply:
The idea is that `ViewDashboard` will have all the providers, `DashboardApp` is where we have local states, and `Dashboard` is just a component. I think it's also confusing how we have two ViewDashboard files.

i'm open to new name proposals :)